service/main.ml

```diff
@@ -40,8 +40,6 @@ module Metrics = struct
     Gauge.set (master "active") (float_of_int active)
 end
 
-let solver = Ocaml_ci.Solver_pool.spawn_local ()
-
 let () =
```

Comment:
Same here - either shift it into `main` or put it into a function which is called from main?